projects/ktlint.github.io/package.yml

```diff
@@ -1,18 +1,14 @@
 distributable:
-  url: https://github.com/pinterest/ktlint/releases/download/{{version}}/ktlint-{{version}}.zip
+  url: https://github.com/pinterest/ktlint/releases/download/{{version}}/ktlint
```

Comment:
what is the different between these archives? i'm not sure our tooling knows how to unpack an archive without an extension.

Reply:
I'll just leave it as a zip then. Probably the people depending on versions before 0.49.1 are few.

Reply:
zero people would be ideal, but i'm not naive enough to think it. ;)